src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {
+  AiOutlineHeart,
+  AiOutlineSearch,
+  AiOutlineShoppingCart,
+} from "react-icons/ai";
+import { Link } from "react-router-dom";
+import "./Header.css";
+
+const Header = () => {
+  const [sticky, setSticky] = React.useState(false);
```

Comment:
add proper line break for separating blocks of code to make this code more readable.